apps/system/js/app_chrome.js

```diff
@@ -137,11 +133,15 @@
   AppChrome.prototype.view = function an_view() {
     var className = this.CLASS_NAME + this.instanceID;
 
-    return `<div class="chrome" id="${className}">
+    return `<div class="chrome chrome-plain" id="${className}">
```

Comment:
"chrome-plain" is the class for this app chrome flavor. "chrome-simple" could work too.